static/assets/governance/code-of-conduct/CODE_OF_COLLAB.md

```diff
@@ -0,0 +1,80 @@
+## Code of Conduct for Collaboration
+
+### Overview
+This document outlines the expectations for collaboration, contribution, and interaction with this project. It is designed to balance the needs of the open-source community with the realities of scientific research, including publishing, funding, and achieving scientific impact. 
+
+---
+
+### **Our Metrics of Success**
+As a project developed by scientific researchers, our primary metrics of success are:
+1. **Achieving scientific and societal impact.**
+2. **Publishing articles in peer-reviewed journals.**
+3. **Gathering funding to support ongoing research and development.**
+
+We strive to balance the demands of open-source development with these goals. Sustainable collaboration is key to achieving these aims.
+
+---
+
+### **Open Science Commitment**
+This project is committed to open science, transparency, and community-driven development, in alignment with initiatives like **NASA's TOPS** (Transform to Open Science). However, open-source development requires significant time and effort, and we want to ensure all collaborators understand the expectations involved.
+
+#### What We Offer:
+1. **Access to cutting-edge research and software** for community use.
+2. **Openness in sharing methods and ideas** while still competing in scientific research.
+3. **Opportunities for collaboration** that align with our metrics of success.
+
+#### What We Expect from You:
+1. **Respect for Our Time and Resources**  
+   Open-source software does not mean free support. We ask that you respect the fact that developing and maintaining software, alongside research, is a time-intensive process. Please make your queries clear and concise to help us assist you efficiently.
+
+2. **Collaborate, Don’t Compete**  
+   Open science thrives on collaboration. We encourage you to collaborate with us, rather than compete against us, when utilizing our software. If our tools are useful for your research, consider working with us directly to co-author papers, share data, or contribute to further developments.
+
+3. **Contribute to Mutual Success**  
+   Our open-source work is designed to benefit both you and the scientific community. If you find our software valuable:
+   - **Cite our work** when using it in research.
```

Comment:
It might be a good idea to link to a citation document here. Should we include, as part of this guide, a template for a citation.MD? If not here might be a good idea to put it someplace else. Get a provides a couple of examples. Perhaps we can include this as a template.

https://docs.github.com/en/repositories/managing-your-repositorys-settings-and-features/customizing-your-repository/about-citation-files